### DISCUSSION TITLE
Cannot call rest api

### DISCUSSION THREAD

**Discussion by aada** [shlinkio/discussions#116] [Help wanted] [answered]
I have installed shlink using docker compose.
When I try to make api requests to Shlink rest api using curl
```
 curl -X 'GET' \
  https://hostname/rest/v3/short-urls \
  -H 'accept: application/json' \
  -H 'X-Api-Key: {key}'
```
I get the following response
```
{"title":"Not Found","type":"NOT_FOUND","status":404,"detail":"Cannot GET https://hostname/v3/short-urls!"}
```
I'm not sure what I'm doing wrong.

**Comment by aada** [comment]
I checked v2 and it works 😅

**Reply by acelaya** [reply, marked as answer]
The API v3 was introduced in Shlink 3.3.0. Any earlier version will return a 404 when using that API version.
